test/blackbox-tests/test-cases/melange/copy-files-include-subdirs.t

```diff
@@ -51,3 +51,4 @@ Now add include_subdirs unqualified to show issue
   $ node $src
   hello from file
   
+
```

Comment:
Why the extra lines?

Reply:
it's probably output from the file we print to stdout.